gcs/CONFIGURATION.md

```diff
@@ -153,14 +153,13 @@ provider.
     The implementation of the AccessTokenProvider interface used for GCS
     Connector.
     
-*   `fs.gs.refresh.access-token.per.request.enable` (default: `false`)
-
-    Whether to generate a new access token for each GCS request.
-    Note: When setting this to true, you have to have an AccessTokenProvider 
-    implementation provided. When using `refresh.access-token.per.request.enable`
-    The low level HTTP retry will be turned off.
-    Note: When this flag is set to true, it takes precedency over 
-    `fs.gs.authorization.handler.impl`
+*   `fs.gs.authorization.mode` (default: `GENERIC`)
+
+    The mode to define how an Access Token is Generated. Possible values are:
+    `Generic`: Generates a generic token, i.e. not aware of request object.
```

Comment:
Upper case, also, how about the word 'BASIC' - which is already used by the 'fs.gs.outputstream.type' property.

Reply:
hmm, the way I'm thinking about it is that it's a token that works for any requests... so basically Generic. Basic could work, I'm not sure if that's more descriptive. What do you think?

Reply:
I was thinking more about the use case, like "basic use case". I don't feel too strong about this though.

Reply:
fair enough. I will rename to basic as it's more abstract and make it easier to change default implementation if we decided to.